src/core/server/types.ts

```diff
@@ -22,3 +22,4 @@ export { PluginOpaqueId } from './plugins/types';
 export * from './saved_objects/types';
 export * from './ui_settings/types';
 export { EnvironmentMode, PackageInfo } from './config/types';
+export { CspConfig } from './csp';
```

Comment:
Not sure about this one, but: `CspConfig` is actually a concrete class, not a proper type. We might want to hide the actual implementation when exporting (at least in `server/types` as this will be imported by browser, but probably in `index` too for consistency)?